flink-runtime/src/main/java/org/apache/flink/runtime/leaderelection/ZooKeeperLeaderElectionService.java

```diff
@@ -411,8 +411,7 @@ protected void handleStateChange(ConnectionState newState) {
 				LOG.debug("Connected to ZooKeeper quorum. Leader election can start.");
 				break;
 			case SUSPENDED:
-				LOG.warn("Connection to ZooKeeper suspended. The contender " + leaderContender.getDescription()
-					+ " no longer participates in the leader election.");
+				LOG.warn("Connection to ZooKeeper suspended, waiting for reconnection.");
```

Comment:
this isn't necessarily true, is it?

Reply:
Actually since Curator already logs ConnectionState change, I'd prefer remove these logs.

For the specific question, if we always turn on the feature, the contender is waiting for reconnection, but it can success(RECONNECTED) or fail(LOST).